test/ParallelIOTest.cpp

```diff
@@ -175,6 +175,10 @@ TEST_CASE( "hdf5_write_test_zero_extent", "[parallel][hdf5]" )
     write_test_zero_extent( "h5", true );
 }
 
+TEST_CASE( "hdf5_write_test_skip_chunk", "[parallel][hdf5]" )
+{
+    write_test_zero_extent( "h5", false );
```

Comment:
according to @RemiLehe's tests these days, only HDF5 seems to be problematic.
So let's expose `H5FD_MPIO_INDEPENDENT` via runtime options #569

Reply:
funny, ADIOS1 BP works for me locally but fails in CI. Could be still a bug in there.

Reply:
potentially exclude this test for now, see above.